apps/web/layouts/default.vue

```diff
@@ -31,7 +31,39 @@
           />
         </template>
       </SfButton>
+      <SfDropdown v-model="isAccountDropdownOpen" placement="bottom-end">
+        <template #trigger>
+          <SfButton
+            variant="tertiary"
+            class="relative text-white hover:text-white active:text-white hover:bg-primary-800 active:bg-primary-900 rounded-md"
+            :class="{ 'bg-primary-900': isAccountDropdownOpen }"
+            @click="accountDropdownToggle()"
+          >
+            <template #prefix><SfIconPerson /></template>
+            {{ account?.firstName }}
+          </SfButton>
+        </template>
+        <ul class="rounded bg-white shadow-md border border-neutral-100 text-neutral-900 min-w-[152px] py-2">
+          <li v-for="{ label, link } in accountDropdown" :key="label">
+            <template v-if="label === $t('account.logout')">
+              <UiDivider class="my-2" />
+              <SfListItem @click="accountDropdownToggle()">{{ label }}</SfListItem>
+            </template>
+            <SfListItem v-else :tag="NuxtLink" :href="link" :class="{ 'bg-neutral-200': $route.path === link }">
+              {{ label }}
+            </SfListItem>
+          </li>
+        </ul>
+      </SfDropdown>
     </nav>
+    <SfButton
+      variant="tertiary"
+      class="relative text-white hover:text-white active:text-white hover:bg-primary-800 active:bg-primary-900 rounded-md md:hidden"
+      square
+      @click="searchModalOpen"
+    >
+      <template #prefix><SfIconSearch /></template>
```

Comment:
Suggestion:
In SFUI2 docs we propose usage in default slot when icon-only button is [rendered](https://docs.storefrontui.io/v2/vue/components/button.html#usage)
```suggestion
      <SfIconSearch />
```

Reply:
https://github.com/vuestorefront/storefront-nuxt3-boilerplate/pull/76/commits/3e2869c1e292120e18692c151115bf5852f5d475